model/accounts_mgmt/v1/root_resource.model

```diff
@@ -148,4 +148,9 @@ resource Root {
 	locator QuotaAuthorizations {
 		target QuotaAuthorizations
 	}
+
+	// Reference to the resource that manages billing models.
+    locator BillingModels {
+    	target BillingModelItem
```

Comment:
You can not target a type model. 

The locator and the target needs to be a resource type defining methods. 

This is the build error you are getting, because the target is not properly defined. You need to update this to `BillingModels`

Reply:
Updated to BillingModels